app/config/FrontendAppConfig.scala

```diff
@@ -115,7 +107,7 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
       case C79Certificate => c79ReturnLink
       case SecurityStatement => adjustmentsReturnLink
       case PostponedVATStatement => postponedVatReturnLink
-      case CashStatement => cashStatementReturnLink
+      case CDSCashAccount => cashStatementReturnLink
```

Comment:
Could you please add a test case for this change?

Reply:
Added tests for this change